tags.md

```diff
@@ -0,0 +1,75 @@
+# Audience
+## Who is our audience (hypothesis)
+* expats/immigrants: people who moved to Porto and looking for the places outside of their homes where they can have quality time
+* curious locals: local crowd that is hungry for something new and good, usually those who speak english well, hang out with expats and traveled outside of Porto
+
+# Tags
+Tags below are required for the audience of the Porto Places to filter and find the place they want to go.
+
+## Category
+Identifies category of the place:
+* food and drink: place to eat and/or drink
+* spot: place to hangout and chill
+
+## Food and drink
+Tags related specifically to the food and drinks offering at places.
+
+### Food and drink type
+* restaurant: full meal, food-oriented
+* tasca: local tavern
+* bakery: pastry and stuff
+* bar: drink-oriented
+
+### Cuisine (if applicable)
+Cuisine type of a place: vietnamese, chinese, asian (do we need generic? maybe pan-asian then?), portuguese, mexican, ukrainian, vegetarian, fusion etc
+
+### Food flair (optional)
+Specific type of food that people may get, for example:
+* burgers
+* wraps (doner, tacos, gyros, pita)
+* pizza
+* sushi
+* pastry
+* francesinha
+* steaks (churrasco) etc
+
+### Drink flair
+Specific type of drink you may get, for example:
+* cocktail
+* beer
+* wine
+* natural wine
+
+## Special features (if applicable)
+Identifies features of the place:
+* view: great scenery or city/landscape
+* outdoor seating: terrace or place to seat on fresh air
+* live performance: live music, performances etc
+* pet-friendly: welcomes pets or your children
+* good for dates: offers good setting for dates
+* lgbtq+-friendly: as it sounds
+* board games friendly: for those who like it
+
+## Local experience (if applicable)
+Identifies how local it is:
+* hidden gem: even locals are not aware of this place much
+* touristic: may be crowded with tourists
+* local: mostly locals there
+> Note: we could also add one more category that is neutral here
+
+## Additional tags
+> Note: BB is still not sure about these
```

Comment:
For work time its probably ok, but I'm not sure about "best times to visit". I've also been wondering if it is good to put something like "happy hours" in the tag or it is better to put such info in the description of the venue

Reply:
I will remove it when get back to the laptop. Agree. Ruck it